drivers/src/soc_ifc.rs

```diff
@@ -15,6 +15,8 @@ Abstract:
 use caliptra_registers::soc_ifc;
 use caliptra_registers::soc_ifc::enums::DeviceLifecycleE;
 
+use crate::FuseBank;
+
 /// Device Life Cycle State
 #[derive(Debug, Copy, Clone, PartialEq, Eq)]
 pub enum Lifecycle {
```

Comment:
Perhaps not part of the scope of the PR, but some drivers are enum types and others are structs. Is there a reason for this?

Reply:
Lifecycle isn't a driver, just a enum with 4 variants.